src/librustdoc/clean/utils.rs

```diff
@@ -484,15 +484,14 @@ crate fn resolve_type(cx: &DocContext<'_>, path: Path, id: hir::HirId) -> Type {
 
 crate fn get_auto_trait_and_blanket_impls(
     cx: &DocContext<'tcx>,
-    ty: Ty<'tcx>,
-    param_env_def_id: DefId,
+    def_id: DefId,
```

Comment:
Should this be `item_def_id` for consistency with the other places?

Reply:
Good idea, done.